package.json

```diff
@@ -17,6 +17,7 @@
     "build:hooks": "microbundle build --raw --cwd hooks",
     "build:test-utils": "microbundle build --raw --cwd test-utils",
     "build:compat": "microbundle build --raw --cwd compat --globals 'preact/hooks=preactHooks'",
+    "postbuild": "cp -R test-utils/ compat/test-utils",
```

Comment:
Maybe we can make it easier for us by just creating a file at `compat/test-utils.js` and populate it with:

```js
module.exports = require("preact/test-utils");
```

Reply:
+1 to this - I'd be worried about breaking import paths when copying things around.